script.js

```diff
@@ -0,0 +1,21 @@
+var playerX = 'X';
+var playerO = 'O'; // creating both players
+var next = '';
+var tabled = $('td'); // get td link from html page
+var playerturn = $('h2'); // get the player in play to display
+var reset = false;
+var restart =$('#reset')
+
+for (var i = 0; i < tabled.length; i++ ) {
+	$('td').on('click', function player1(event) {
```

Comment:
When you add event listeners to an array in jQuery like this, you don't need to loop through the array - jQuery can add to all of them at once.